src/Symfony/Component/PasswordHasher/Hasher/PasswordHasherFactory.php

```diff
@@ -200,7 +200,7 @@ private function getHasherConfigFromAlgorithm(array $config): array
                     $config['algorithm'] = 'native';
                     $config['native_algorithm'] = \PASSWORD_ARGON2ID;
                 } else {
-                    throw new LogicException(sprintf('Algorithm "argon2id" is not available. Either use "%s", upgrade to PHP 7.3+ or use libsodium 1.0.15+ instead.', \defined('PASSWORD_ARGON2I') || $hasSodium ? 'argon2i", "auto' : 'auto'));
+                    throw new LogicException(sprintf('Algorithm "argon2id" is not available; use "%s" or ext-sodium 1.0.15+ instead.', \defined('PASSWORD_ARGON2I') || $hasSodium ? 'argon2i", "auto' : 'auto'));
```

Comment:
IIRC this limitation is is really about libsodium itself, not the pecl. https://github.com/jedisct1/libsodium/releases/tag/1.0.15

Reply:
fixed thanks